lib/org/chromium/webidl/PipelineRunner.js

```diff
@@ -35,12 +33,18 @@ foam.CLASS({
     function fmtErrorMsg(msg) {
       return new Error(`${this.cls_.name}: ${msg}`);
     },
-    function validateMessage(message) {
-      if (!this.PipelineMessage.isInstance(message)) {
-        var errorMsg = this.fmtErrorMsg(this.INVALID_ARG);
+    function validateMessage(msg) {
+      if (!this.inputType.isInstance(msg)) {
+        var errorMsg = this.fmtErrorMsg(
+            `run() expects a ${this.inputType.name} object!`);
         this.error(errorMsg);
         return errorMsg;
       }
     },
+    function output(msg) {
+      if (!this.outputType.isInstance(msg))
+        console.warn(`${this.cls_.name}: Message does not match outputType!`);
```

Comment:
nit: For symmetry with input error: `${this.cls_.id}.output(): Message does not match outputType!`